Makefile

```diff
@@ -107,7 +107,7 @@ container-build: module-check
 container-serve: module-check ## Boot the development server using container.
 	$(CONTAINER_RUN) --cap-drop=ALL --cap-add=AUDIT_WRITE --read-only \
 		--mount type=tmpfs,destination=/tmp,tmpfs-mode=01777 -p 1313:1313 $(CONTAINER_IMAGE) \
-		hugo server --buildFuture --environment development --bind 0.0.0.0 --destination /tmp/public --cleanDestinationDir --noBuildLock
+		hugo server --buildDrafts --buildFuture --environment development --bind 0.0.0.0 --destination /tmp/public --cleanDestinationDir --noBuildLock
```

Comment:
This change looks good! 

Would it be possible to apply a similar update to the [`serve`](https://github.com/kubernetes/website/blob/cee8816e934584782f1e6364bba6244ea5f9da49/Makefile#L58-L59) Makefile target as well? It’s used for testing the site locally outside of the container, so it might help improve consistency with container preview method.
 
https://github.com/kubernetes/website/blob/cee8816e934584782f1e6364bba6244ea5f9da49/Makefile#L58-L59

Reply:
Maybe then also at line 99 to apply the change uniformly to every CMD

Reply:
See line 104 @mbianchidev